app/main.py

```diff
@@ -1,6 +1,33 @@
+import requests
+import os
+
+API_KEY = os.getenv("API_KEY")
+
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        URL,
+        params={
+            "key": API_KEY,
+            "q": CITY,
+        }
+    )
+
+    if response.status_code != 200:
+        print(f"Can't get data from Weather API. "
+              f"Status code: {response.status_code}")
+
+    data = response.json()
+    print(
```

Comment:
Your program will fail if the status code is not 200. It would be better to retrieve the data within a block of code that checks if the status code is 200.